test/e2e/components/vl-infoblock.js

```diff
@@ -0,0 +1,63 @@
+const { VlElement } = require('vl-ui-core');
+const { VlIcon } = require('vl-ui-icon');
+
+class VlInfoblock extends VlElement {
+    constructor(driver, selector) {
+        super(driver, selector);
+    }
+
+    async getText() {
+        const element = await this._getContent();
+        return this.driver.executeScript('return arguments[0].innerText', element);
+    }
+
+    async getTitle() {
+        const element = await this._getContent();
+        const slots = await element.findElements(By.css('[slot]'));
+        if (slots.length > 0) {
+            return this.getText();
+        } else {
+            return this.getAttribute('title');
+        }
+    }
+
+    async _getContent() {
+        return await this.shadowRoot.findElement(By.css('#infoblock_content'));
+    }
+
+    async _getType() {
+        return this.getAttribute('type');
+    }
+
+    async isContact() {
+        return (await this._getType()) == 'contact';
+    }
+
+    async isPublication() {
+        return (await this._getType()) == 'publications';
+    }
+
+    async isFaq() {
+        return (await this._getType()) == 'faq';
+    }
+
+    async isNews() {
+        return (await this._getType()) == 'news';
+    }
+
+    async isTimeline() {
+        return (await this._getType()) == 'timeline';
+    }
+
+    async isQuestion() {
+        return (await this._getType()) == 'question';
+    }
+
+    async isCalendar() {
+        return (await this.getAttribute('icon')) == 'calendar';
+    }
```

Comment:
da is misschien niet per se nodig ...

Reply:
Is voor mij ook niet perse nodig maar heb liever een iets uitgebreidere API omdat we nu nog niet weten wat de afnemers allemaal gaan nodig hebben. Eindbeslissing laat ik aan @coemans over.

Reply:
Ik zal enkele testen toevoegen zodat deze functies meteen gebruikt worden en we zeker zijn dat ze goed werken.